db/schema.rb

```diff
@@ -0,0 +1,61 @@
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# Note that this schema.rb definition is the authoritative source for your
+# database schema. If you need to create the application database on another
+# system, you should be using db:schema:load, not running all the migrations
+# from scratch. The latter is a flawed and unsustainable approach (the more migrations
+# you'll amass, the slower it'll run and the greater likelihood for issues).
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema.define(version: 20180409235316) do
```

Comment:
The database seems to be missing proper indexing and constraints. Was that intentional?

Reply:
Not intentional. More of an oversight. Ideally I would index the join columns/columns that can be searched on/columns that can be sorted on as well as enforcing required fields with ```null: false``` to not allow ```NULL``` values.